apps/css-workshop/src/components/Badge.astro

```diff
@@ -0,0 +1,32 @@
+---
+type Props = {
+  backgroundColor?:
+    | 'skyblue'
+    | 'celery'
+    | 'froly'
+    | 'steelblue'
+    | 'sunglow'
+    | 'seabuckthorn'
+    | 'montecarlo'
+    | 'poloblue'
+    | 'bouquet'
+    | 'ash'
+    | 'oak';
+  status?: 'informational' | 'positive' | 'warning' | 'negative';
+} & astroHTML.JSX.HTMLAttributes;
+
+const { class: className, backgroundColor, status, ...props } = Astro.props;
+---
+
+<span
+  class:list={['iui-badge', className]}
+  style={{
+    '--iui-badge-background-color': backgroundColor
+      ? `var(--iui-color-background-${backgroundColor})`
+      : undefined,
```

Comment:
This is just a [ternary](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator) expression for conditionally applying the inline CSS variable style only when `backgroundColor` is ["truthy"](https://developer.mozilla.org/en-US/docs/Glossary/Truthy)